bot/bot.py

```diff
@@ -10,6 +13,13 @@
 class SirRobin(commands.Bot):
     """Sir-Robin core."""
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.name = "Sir-Robin"
+        self._guild_available = asyncio.Event()
+        create_task(self.check_channels(), event_loop=self.loop)
+        create_task(self.send_log(self.name, "Connected!"), event_loop=self.loop)
```

Comment:
We already have the name in constants.
```suggestion
        self._guild_available = asyncio.Event()
        create_task(self.check_channels(), event_loop=self.loop)
        create_task(self.send_log(constants.Client.name, "Connected!"), event_loop=self.loop)
```